### PR TITLE
Add DEPENDENCY_MANIFEST_OF relationship

### PR DESCRIPTION
**Pull request by tsteenbe**
Relates-to: #174.

Signed-off-by: Thomas Steenbergen <thomas.steenbergen@here.com>

### CODE REVIEW

File: chapters/7-relationships-between-SPDX-elements.md [hunk 1]
```diff
@@ -10,6 +10,7 @@
 | DESCRIBED_BY           | Is to be used when SPDXRef-A is described by SPDXREF-Document                                         | The package ‘WildFly’ is described by SPDX document `Wildfly.spdx`. |
 | CONTAINS               | Is to be used when SPDXRef-A contains SPDXRef-B.                                                      | An ARCHIVE file `bar.tgz` contains a SOURCE file `foo.c`. |
 | CONTAINED_BY           | Is to be used when SPDXRef-A is contained by SPDXRef-B.                                               | A SOURCE file `foo.c` is contained by ARCHIVE file `bar.tgz` |
+| DEPENDENCY_MANIFEST_OF | Is to be used when SPDXRef-A is a manifest file that lists a set of dependencies for SPDXRef-B        | A file package.json is the dependency manifest of a package foo. Note that only one manifest should be used to defined the same dependency graph. |
```
Comment (swinslow) [inline chapters/7-relationships-between-SPDX-elements.md]: last sentence, "defined" => "define"

Comment (tsteenbe) [reply chapters/7-relationships-between-SPDX-elements.md]: Fixed

**Comment by swinslow** [comment]
:tada:

**Comment by swinslow** [comment]
all backticks now approved  +1

**Comment by kestewart** [comment]
Typos (adding backticks) spotted by Alexios fixed during the call.    All approved now.

**Pull request merged**